engine/core/IgeViewport.ts

```diff
@@ -28,12 +28,9 @@ export interface IgeViewportOptions {
 export class IgeViewport extends IgeUiEntity implements IgeCanRegisterById {
 	classId = "IgeViewport";
 	IgeViewport = true;
-	_idRegistered: boolean = false;
```

Comment:
I think this popped up when I tried to build my game after referencing IGE, it threw an error about shadowing essentially. I'm also using Intellij and it didn't seem to mind. Not sure why tsc didn't pick it up, It could be a difference in tsconfig settings maybe